api/client/samples/crop_models/sao_paulo_sugar_content/Sao_Paulo_brazil_sugar_content_modeling.ipynb

```diff
@@ -0,0 +1,3628 @@
+{
```

Comment:
<p>mention that we aggregate temperature as a mean (unlike rainfall which is a sum)</p><p> </p><p>again statistically the difference is just a constant so no diff from a modellign point of view.  but physically we should be accurate (in case e.g. time period changes)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
<p>Done</p>